Formula/epa-ng.rb

```diff
@@ -0,0 +1,24 @@
+class EpaNg < Formula
+  desc "Massively parallel phylogenetic placement of genetic sequences"
+  homepage "https://github.com/Pbdas/epa-ng"
+  url "https://github.com/Pbdas/epa-ng/archive/v0.3.5.tar.gz"
+  sha256 "0aa2aa9ffad0d713c42ea43330798a4d701ece883ced617da07218b7c697df41"
+
+  depends_on "cmake" => :build
+  unless OS.mac?
+    depends_on "bison" => :build
+    depends_on "flex" => :build
+  end
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+    end
+    bin.install Dir["bin/epa-ng*"]
+  end
+
+  test do
+    system "#{bin}/epa-ng", "--version"
```

Comment:
```suggestion
    assert_match "Usage", shell_output("#{bin}/epa-ng --help")
```
Modify as necessary to make the test pass.